java/sage/FFMPEGTranscoder.java

```diff
@@ -635,9 +635,29 @@ else if ("container".equals(propName))
           catch (NumberFormatException e)
           {}
         }
-
-        xcodeParams = "-f " + f + " -vcodec " + vcodec + " -s " + s + " -ac " + ac + " -g " + g + " -bf " + bf + (deinterlace ? " -deinterlace " : "") +
-            " -acodec " + acodec + " -r " + r + " -b " + b + " -ar " + ar + " -ab " + ab + " -packetsize " + packetsize;
+        
+        xcodeParams = "-f " + f;
+        
+        if(vcodec.equals("COPY"))
+        {
+          xcodeParams += " -vcodec copy";
+        }
+        else
+        {
+          xcodeParams += " -vcodec " + vcodec + " -s " + s + " -ac " + ac + " -g " + g + " -bf " + bf + (deinterlace ? " -deinterlace " : "");
```

Comment:
"ac" is audio channels, so that should go with the audio codec. "b" is video bitrate, so that should go with video. "r" is the frame rate, so that should go with video.

Reply:
Sorry about that.   I am glad that you caught that.  I only tested with audio/video copy together.  I did not test with a copy on  only audio or video.  I think I got it correct with the latest commit.